Transfer/Sources/CreateTransferPresenter.swift

```diff
@@ -389,8 +389,20 @@ final class CreateTransferPresenter {
     }
 
     func resetErrorMessagesForAllSections() {
-        sectionData.forEach { $0.errorMessage = nil }
-        CreateTransferController.FooterSection.allCases.forEach({ view?.updateFooter(for: $0) })
+        if sectionData.contains(where: { $0.errorMessage != nil }) {
+            sectionData.filter({ $0.errorMessage != nil }).forEach({
+                $0.errorMessage = nil
+                switch $0.createTransferSectionHeader {
+                case .amount: view?.updateFooter(for: .amount)
+
```

Comment:
Empty line looks odd, kindly remove it. Also generally a good idea to add new lines after each case - looks cleaner IMO
```
case .amount: 
        view?.updateFooter(for: .amount)
```

Reply:
Sure